File: src/main/java/me/wuzzyxy/dynamicmarket/configs/ReportSettings.java
package me.wuzzyxy.dynamicmarket.configs;

/***
 * Just the report knobs, so MarketReport does not need a whole PluginConfig — which can
 * only be built from a live Bukkit config and would drag the server into every test.
 */
public record ReportSettings(
        int windowHours,
        double minChange,
        String moverUp,
        String moverDown,
        String trendUp,
        String trendDown,
        String trendFlat,
        String empty
) {}

File: src/main/kotlin/me/wuzzyxy/dynamicmarket/configs/ReportSettings.kt
package me.wuzzyxy.dynamicmarket.configs

/***
 * Just the report knobs, so MarketReport does not need a whole PluginConfig — which can
 * only be built from a live Bukkit config and would drag the server into every test.
 */
data class ReportSettings(
    val windowHours: Int,
    val minChange: Double,
    val moverUp: String,
    val moverDown: String,
    val trendUp: String,
    val trendDown: String,
    val trendFlat: String,
    val empty: String,
)
